### PR TITLE
Disable hue light controls when in sync mode

### PR DESCRIPTION
**Pull request by bflorian**
Update Philips Hue light device profiles to reference device configurations that disable the light controls when it is in sync mode. This PR should not be merged until the device configurations have been deployed.

### CODE REVIEW

File: drivers/SmartThings/philips-hue/profiles/legacy-color.yml
Comment (dljsjr) [inline drivers/SmartThings/philips-hue/profiles/legacy-color.yml:9]: Since the custom presentation isn't visible in this changeset, I wanted to make a visibility comment here to call out that these embedded device configs are removed in this PR because they're rolled in to the custom presentation upon which this PR depends.

**Review by greens — APPROVED** [review]

**Comment by dljsjr** [comment]
Added `do not merge`, it was requested that we wait for some mobile app and infrastructure work before merging this.

**Review by cjswedes — APPROVED** [review]

**Review by mantiger — COMMENTED** [review]
This code is the last code to finalize hue music sync.
It should be included in R2 timeline so please merge it as soon as possible.

**Pull request merged**
